coding-for-reusability/Exercise6/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "training",
+  "version": "0.0.1",
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  }
+}
```

Comment:
I think this needs to be in the `problem` folder.

Here’s what I get right now: https://codesandbox.io/s/github/bitovi/trainings/tree/coding-for-reusability/coding-for-reusability/Exercise6/problem

Reply:
Ah, Apparently it makes a difference how the link gets opened.